pkg/controllers/repositoryserver/utils.go

```diff
@@ -253,8 +253,8 @@ func getCondition(status metav1.ConditionStatus, reason string, message string,
 	}
 }
 
-func getVolumes(ctx context.Context, cli kubernetes.Interface, secret *corev1.Secret, namespace string) (map[string]string, error) {
-	vols := make(map[string]string, 0)
+func getVolumes(ctx context.Context, cli kubernetes.Interface, secret *corev1.Secret, namespace string) (map[string]kube.VolumeMountOptions, error) {
```

Comment:
```suggestion
func getVolumes(
	ctx context.Context,
	cli kubernetes.Interface,
	secret *corev1.Secret,
	namespace string,
) (map[string]kube.VolumeMountOptions, error) {
```
Improves readability

Reply:
fixed